GitCommands/Config/ConfigFile.cs

```diff
@@ -10,8 +10,7 @@ namespace GitCommands.Config
 {
     public class ConfigFile
     {
-        private readonly string _fileName;
-        public string FileName { get { return _fileName; } }
+        public string FileName { get; }
```

Comment:
:memo: Using getter-only auto-properties should be sent as a separate pull request